java/src/main/java/org/digma/intellij/plugin/idea/psi/java/IEndpointDiscovery.java

```diff
@@ -15,8 +15,13 @@ public interface IEndpointDiscovery {
 
```

Comment:
We don't use I as prefix of interfaces names? and its not common in java. why start a new convention?